app/timetables/admin.py

```diff
@@ -1,7 +1,7 @@
 from django.contrib import admin
 
 from .models import (Weekday, MealOption, Course, Meal, Timetable,
```

Comment:
@andela-eisaac I prefer it that way as well. @andela-kanyanwu and @andela-uawili Please, what's your take on this as you prefer this current implementation.